blog/steering/governance-changes.md

```diff
@@ -0,0 +1,87 @@
+Author handle: https://github.com/pmorie
+Description: Summary of governance changes to steering and trademark committees.
+Folder with media files: 'N/A'
+Blog URL: ''
+Labels: Articles
+Reviewers: ''
+date: 2020-09-30
+description: "Knative Project Governance Changes"
+type: "blog"
+
+---
+| Reviewer  | Date | Approval |
+| ------------- | ------------- | ------------- |
+| <!-- Your Github handle here --> |   |  |
+
+
+One of the long-standing [asks](https://github.com/knative/community/issues/33)
+within the Knative community has been for the Steering Committee to clarify its
+charter and how the committee is composed. I'm happy to be able to say that we
+now have a new [Steering Committee
+charter](https://github.com/knative/community/pull/277) which, in my opinion,
+delivers on this ask. Community participation was central to working through the
+process to get to this new charter, and I first and foremost want to say a
+sincere THANK YOU to everyone in our community that participated in our public
+discussions on this topic. I don't think it is an exaggeration to say that the
+entire community was involved in this effort, or that the community
+participation helped to influence the direction the discussions went.
+
+Let me summarize the meaning of the changes, and then go into some detail:
+
+- There will be elections for Steering Committee members
+- Steering committee members serve as individuals rather than representing their employer
+- No vendor will be allowed to have a majority of seats in the Steering Committee
+- We added the new Knative Trademark Committee (KTC) to handle matters related to the trademark
+- There is a transition plan from the current bootstrap scheme to the new governance scheme
+
+Those of you who love committees will be pleased that we have increased the
+committee count within the project by one. Those of you who don't love
+committees for their own sake may be asking what the purpose of such a committee
+is; luckily, there is a clear and sensible answer. During the discussions around
+new governance, it became apparent that there was a clear separation of concerns
+between the specific vendor needs around trademark, "core" scope (for lack of a
+better term) and conformance, and the broader community needs around everything
+else Steering does. While we could have created a hybrid structure for Steering,
+we decided it was simpler to just have a vendor committee that focused only on
+scope and conformance, with the Steering committee focusing on the other aspects
+of community growth and project governance.
+
+The KTC will initially include representatives from Google, Red Hat/IBM, and
+VMware. They will need to figure out how new components and features "plug in"
+to the Knative ecosystem in a non-technical sense. They will also help Steering
+settle questions around derivative trademarks and logos, Knative "extensions",
+and other questions about being part of our larger project. Membership to the KTC
```

Comment:
“Membership” is an interesting term since it might imply some monetary contribution, like membership in a club. Of course, it also has the other meaning, elected members. Could be clearer on how membership is attained, renewed, and / or revoked? Just to clarify.